org.sonarlint.eclipse.core/pom.xml

```
@@ -111,7 +111,7 @@
                 <artifactItem>
                   <groupId>org.sonarsource.text</groupId>
                   <artifactId>sonar-text-plugin</artifactId>
-                  <version>2.10.0.2188</version>
+                  <version>2.12.0.2889</version>
```

Bugfix release, I updated the ticket
```suggestion
                  <version>2.12.1.2905</version>
```